ca_description/urdf/create_base.xacro

```diff
@@ -53,7 +54,12 @@
     <xacro:create_wheel prefix="left" y_offset="${wheel_separation / 2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:create_wheel prefix="right" y_offset="${wheel_separation / -2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:caster_wheel/>
-
+    <xacro:color_sensor parent="base_link" reflect="1" prefix="right">
```

Comment:
The idea behind passing the origin block is to avoid using `reflect` and `prefix`. Check how you can do to remove them.

Reply:
Tip: with this approach, both origin blocks should be different.

Reply:
I removed the `reflect` param, i think the `prefix` one is pretty awesome, i can remove it if you want though. Please let me know

Reply:
If you can set the origin when you're declaring the macro, this parameter shouldn't be necessary, right?